packages/docs/celo-codebase/protocol/identity/accounts.md

```diff
@@ -0,0 +1,75 @@
+
+> Crypto is a tool for turning a whole swathe of problems into key management problems. Key management problems are way harder than (virtually all) cryptographers think.
+
+To help with this, we have various mechanisms to reduce the impact of the loss or compromise of keys, mainly through our `Accounts` smart contract. On it, accounts can authorize other keys to perform certain actions on behalf of the account. Keys that need to be accessed frequently (e.g. for signing blocks) are at greater risk of being compromised, and thus have more limited permissions, while keys that need to be accessed infrequently (e.g. for locking Celo Gold) are less onerous to store securely, and thus have more expansive permissions. Below is a summary of the various keys that are used in the Celo network, and a description of their permissions.
+
+| Name of the key        | Purpose                                                                                                                                                                                                                                                               |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Account key            | This is the key with the highest level of permissions, and is thus the most sensitive. It can be used to lock and unlock Celo Gold, and authorize vote, validator, and attestation keys. Note that the account key also has all of the permissions of the other keys. |
+| Validator signer key   | This is the key that has permission to register and manage a Validator or Validator Group, and participate in BFT consensus.                                                                                                                                          |
+| Vote signer key        | This key can be used to vote in Validator elections and on-chain governance.                                                                                                                                                                                          |
+| Attestation signer key | This key is used to sign attestations in Celo's lightweight identity protocol.                                                                                                                                                                                        |
+
+Note that account and signer keys must be unique and may not be reused.
+
+# Key Rotation
+
+Loss of an authorized key no longer has catastrophic implications, as the Account key can just authorize another key in that case. In general, it is good hygiene to rotate your keys, just like you should rotate your passwords.
```

Comment:
I think we should strengthen the second sentence. (It's also good hygiene to floss, but most reader aren't doing that)
```suggestion
Loss of an authorized key doesn't have to be catastrophic, as the Account key can just authorize another key in that case. It is strongly recommended to regularly rotate keys to limit the impact of an undiscovered compromise.
```